platform/firecamp-ui/src/components/tooltip/ToolTip.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react';
+
+import { MantineColor, Tooltip as MantineToolTip } from '@mantine/core';
+import { ArrowPosition, FloatingPosition } from '@mantine/core/lib/Floating';
+
+interface IToolTip {
+  arrowOffset?: number;
+  arrowPosition?: ArrowPosition;
+  arrowRadius?: number;
+  arrowSize?: number;
+  children: React.ReactNode;
+  closeDelay?: number;
+  color?: MantineColor;
+  disabled?: boolean;
+  events?: {
+    hover: boolean;
+    focus: boolean;
+    touch: boolean;
+  };
+  label: React.ReactNode;
+  multiline?: boolean;
+  offset?: number;
+  openDelay?: number;
+  postition?: FloatingPosition;
```

Comment:
done

Reply:
Hi Updated the tooltip position to right since it looks better than top.

![right position](https://github.com/firecamp-dev/firecamp/assets/53973174/f6ca8b9e-e876-4aee-b0c4-324321b928a2)